azurerm/resource_arm_container_group.go

```diff
@@ -0,0 +1,323 @@
+package azurerm
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/arm/containerinstance"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmContainerGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmContainerGroupCreate,
+		Read:   resourceArmContainerGroupRead,
+		Delete: resourceArmContainerGroupDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"location": locationSchema(),
+
+			"resource_group_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"ip_address_type": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				Default:          "Public",
+				ForceNew:         true,
+				DiffSuppressFunc: ignoreCaseDiffSuppressFunc,
+				ValidateFunc: validation.StringInSlice([]string{
+					"Public",
+				}, true),
+			},
+
+			"os_type": {
+				Type:             schema.TypeString,
+				Required:         true,
+				ForceNew:         true,
+				DiffSuppressFunc: ignoreCaseDiffSuppressFunc,
+				ValidateFunc: validation.StringInSlice([]string{
+					"windows",
+					"linux",
+				}, true),
+			},
+
+			"tags": tagsForceNewSchema(),
+
+			"ip_address": {
+				Type:     schema.TypeString,
+				Computed: true,
+				ForceNew: true,
```

Comment:
given this is Computed, it doesn't need to be ForceNew - so we can remove this

Reply:
Done.